googleads/ad_manager.py

```diff
@@ -897,7 +899,7 @@ def WaitForReport(self, report_job):
 
     while status != 'COMPLETED' and status != 'FAILED':
       _data_downloader_logger.debug('Report job status: %s', status)
-      time.sleep(30)
+      time.sleep(poolTimeSeconds)
```

Comment:
pollTimeSeconds

Reply:
Can you make the variable name "pollTimeSeconds" instead of "poolTimeSeconds"?
```suggestion
      time.sleep(pollTimeSeconds)
```